Sources/Turf/Geometries/Polygon.swift

```diff
@@ -297,4 +297,55 @@ extension Polygon {
             ring[2].longitude == ring[0].longitude
         )
     }
-}
+
+
+    /// Calculates the centroid using the mean of all vertices.
+    /// This lessens the effect of small islands and artifacts when calculating the centroid of a set of polygons.
+    public func centroid() -> LocationCoordinate2D? {
+        let coordinates = outerRing.coordinates.dropLast()
+        let summed = coordinates
+            .reduce(into: LocationCoordinate2D(latitude: 0, longitude: 0)) { acc, next in
+                acc.latitude += next.latitude
+                acc.longitude += next.longitude
```

Comment:
Should we be concerned about features that span the antimeridian? Technically, GeoJSON doesn’t allow a single feature to span the antimeridian, but it’s common (at least among Mapbox products) to represent it as a single feature with coordinates that extend beyond ±180°. If this implementation doesn’t handle that case correctly, that’s OK; we can open a separate issue for it.

Reply:
Good point. I don't think it handles that properly. It should check if the final longitude extends beyond ±180° and then snap it back within the allowed range. Agree that it might be better to create a separate issue for that. (Turf.js might also need fixing for that.)

Reply:
The fix would be to wrap the longitude to the ±180° range. Snapping would result in truncating the geometry at the antimeridian, which would be unexpected.

Reply:
Done!